app-misc/openrgb/openrgb-0.8.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 2020-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit flag-o-matic qmake-utils udev xdg-utils
+
+if [[ ${PV} == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI=${EGIT_REPO_URI:-"https://gitlab.com/CalcProgrammer1/OpenRGB"}
+else
+	SRC_URI="https://gitlab.com/CalcProgrammer1/OpenRGB/-/archive/release_${PV}/OpenRGB-release_${PV}.tar.bz2"
+	S="${WORKDIR}/OpenRGB-release_${PV}"
+	KEYWORDS="~amd64 ~x86"
+	PATCHES=( "${FILESDIR}"/OpenRGB-0.5-build-system.patch )
+fi
+
+DESCRIPTION="Open source RGB lighting control"
+HOMEPAGE="https://openrgb.org https://gitlab.com/CalcProgrammer1/OpenRGB/"
+LICENSE="GPL-2"
+# subslot is OPENRGB_PLUGIN_API_VERSION from https://gitlab.com/CalcProgrammer1/OpenRGB/-/blob/master/OpenRGBPluginInterface.h
+SLOT="0/2"
+
+RDEPEND="
+	dev-cpp/cpp-httplib:=
+	dev-libs/hidapi
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtwidgets:5
+	net-libs/mbedtls:=
+	virtual/libusb:1
+"
+DEPEND="
+	${RDEPEND}
+	dev-cpp/nlohmann_json
+	dev-libs/mdns
+"
+BDEPEND="
+	dev-qt/linguist-tools:5
+	virtual/pkgconfig
+"
+
+PATCHES+=(
+	"${FILESDIR}"/OpenRGB-0.7-r1-udev.patch
+)
+
+src_prepare() {
+	default
+	rm -r dependencies/{httplib,hidapi,libusb,mdns,json,mbedtls}* \
+		|| die "Failed to remove unneded deps"
+}
+
+src_configure() {
+	# Some plugins require symbols defined in the main binary.
+	# The official build system bundles OpenRGB as a submodule instead, and
+	# compiles the .cpp file again.
+	append-ldflags -Wl,--export-dynamic
+
+	# > warning: ‘-pipe’ ignored because ‘-save-temps’ specified
+	filter-flags -pipe
+
+	eqmake5 \
+		INCLUDEPATH+="${ESYSROOT}/usr/include/nlohmann" \
+		DEFINES+="OPENRGB_EXTRA_PLUGIN_DIRECTORY=\\\\\"\\\"${EPREFIX}/usr/$(get_libdir)/OpenRGB/plugins\\\\\"\\\"" \
+		LIBS+=-lhttplib
+}
+
+src_install() {
+	emake INSTALL_ROOT="${ED}" install
+
+	dodoc README.md OpenRGB.patch
+
+	udev_dorules 60-openrgb.rules
+	rm -r "${ED}"/usr/lib/udev/ || die
```

Comment:
Btw, what is happening here? I had to remove this line in order to have openrgb detect some `/dev/hidraw` devices. Why do we install this udev file only to delete it again?